NEWS

```diff
@@ -1,3 +1,14 @@
+master
+------
+
+Enhancements
+ - Full contact properties working and saved between Outlooks
```

Comment:
- Missing contact fields are now saved and available when sharing it
  (Missing contact fields here)